services/credential-schema/prisma/schema.prisma

```diff
@@ -0,0 +1,51 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider          = "postgresql"
```

Comment:
move to environment variable.

Reply:
the solution for this will be raised as a separate PR for all three services